CMakeLists.txt

```diff
@@ -1,6 +1,14 @@
 cmake_minimum_required (VERSION 2.8.11)
 project (benchmark)
 
+foreach(p
```

Comment:
this shouldn't be necessary. the rest of the cmake config needs to work without this set, so there's no need to set it.